src/types.tsx

```diff
@@ -179,6 +179,10 @@ export interface ScreenContainerProps extends ViewProps {
    * A prop that gives users an option to switch between using Screens for the navigator (container). All children screens should have the same value of their "enabled" prop as their container.
    */
   enabled?: boolean;
+  /**
+   * A prop to be used in drawer and bottom-tabs navigators for better implementation of ScreenContainer on iOS.
+   */
+  tabsOrDrawer?: boolean;
```

Comment:
I think a prop name like `hasTwoStates` (didn't think a lot about the name) or something like that would be more generic and more appropriate